tools/create_db.py

```diff
@@ -0,0 +1,26 @@
+import psycopg2
```

Comment:
To już zdecydowanie nie jest meijsce na te rzeczy. Albo modul infrastructure, albo jakiś skryp bashowy do tego. Najlepiej docker :P